src/spawn_sync.cc

```diff
@@ -378,8 +378,20 @@ void SyncProcessRunner::RegisterExternalReferences(
 
 void SyncProcessRunner::Spawn(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
+  Local<Context> context = env->context();
+
+  std::string resource = "";
+  if (env->permission()->enabled() && args[0]->IsObject()) {
+    Local<Object> js_options = args[0].As<Object>();
+    Local<Value> js_file;
+    if (js_options->Get(context, env->file_string()).ToLocal(&js_file)) {
+      node::Utf8Value v(env->isolate(), js_file.As<String>());
+      resource = v.ToString();
+    }
```

Comment:
```suggestion
    if (!js_options->Get(context, env->file_string()).ToLocal(&js_file)) {
      return;
    }
    
    node::Utf8Value v(env->isolate(), js_file.As<String>());
    resource = v.ToString();
```

I think if `ToLocal` doesn't work for some reason, an error will be scheduled. Just like in [here](https://github.com/nodejs/node/pull/58758/files#diff-9659f07c469aacdd5e1805014f9220761d8614acad9edeafb29ff402588cbb21R205-R208)